icecreamshop/src/main/java/com/summerschool/icecreamshop/controller/BasketController.java

```diff
@@ -59,4 +54,17 @@ public BasketController(BasketService basketService, ModelMapper modelMapper) {
         Basket updatedBasket = basketService.patch(modelMapper.map(basketDTO,Basket.class),foundBasket);
         return ResponseEntity.ok(modelMapper.map(updatedBasket, BasketDTO.class));
     }
+
+    @GetMapping("")
+    public ResponseEntity<List<BasketProductDTO>> getBasketProducts(
+            @RequestParam(value = "basketId", required = false) Long basketId
```

Comment:
De asemenea, ar fi bine daca parametrul il setam ca Path parameter, in loc de query param